website/docs/FAQ/04-PaymentFAQ.md

```diff
@@ -5,12 +5,18 @@ title: Purchases & Payment Processing
 Please note all orders ship **after** full payment. Orders cannot be split between multiple shipments. To make a change to your order, please email [support@openbci.com](mailto:support@openbci.com) ASAP. We cannot guarantee that we can satisfy the request, but we will do our best if contacted in a timely fashion.
 
 ## How do I request a quotation or an invoice?
+There are two methods of getting a quotation. 
 
-To receive a quote, please make a [customer account](https://shop.openbci.com/account/register) and add your shipping address to your account. You can add your address by clicking “view addresses” on the right hand side of the screen. Then, click “Add a new address”. 
+The first and faster method is to use the self-service quote tool on our [website](https://shop.openbci.com). Select the product you'd like a quote for and use the "Add to Quote" button (below the "Add to cart" button.) You can add multiple items to the quote by clicking "Continue Browsing". Once you have all items required added to the quote, click "Show Quote Cart" to finalize the quote. You will receive an email with the PDF quote. 
+
+The second method-
```

Comment:
[THREE HASHTAGS] Request a quote via email

Reply:
I would change this part that just says "The second method-" . Maybe it can be removed.